fuels-core/src/constants.rs

```diff
@@ -8,8 +8,8 @@ pub const WORD_SIZE: usize = core::mem::size_of::<Word>();
 
 // This constant is used to determine the amount in the 1 UTXO
 // when initializing wallets for now.
-pub const DEFAULT_COIN_AMOUNT: u64 = 1;
-pub const DEFAULT_INITIAL_BALANCE: u64 = 100;
+pub const DEFAULT_COIN_AMOUNT: u64 = 1_000_000;
+pub const DEFAULT_INITIAL_BALANCE: u64 = 1_000_000_000;
```

Comment:
Technically orthogonal to this issue, but what's the different between coin amount and initial balance?

Reply:
One is for testing, the other is a temporary thing to setup some arbitrary coins to a transaction: https://github.com/FuelLabs/fuels-rs/blob/master/fuels-contract/src/contract.rs#L152-L155

Naming could be improved though, very confusing indeed!

Reply:
If it's for testing then it shouldn't be pub

Reply:
Yeah. @luizstacio do you mind removing `pub` from `DEFAULT_INITIAL_BALANCE`?

Reply:
I add some comments on it but would require more work to remove pub https://github.com/FuelLabs/fuels-rs/issues/180